server/src/main/java/io/littlehorse/server/streams/BackendInternalComms.java

```diff
@@ -19,7 +19,6 @@
 import io.littlehorse.common.LHServerConfig;
 import io.littlehorse.common.Storeable;
 import io.littlehorse.common.exceptions.LHApiException;
-import io.littlehorse.common.exceptions.LHBadRequestError;
```

Comment:
This class should be removed from the code I think. We should have deleted it when we did the GRPC facelift (the horrible dark days when we had our own error code schema :joy: )